js/pptx/smoke/api_drawing/select.js

```diff
@@ -0,0 +1,13 @@
+builder.CreateFile("pptx");
+var oPresentation = Api.GetPresentation();
+var oSlide = oPresentation.GetSlideByIndex(0);
+oSlide.RemoveAllObjects();
+var oDrawing = Api.CreateImage(
+    "https://legacy-api.onlyoffice.com/content/img/docbuilder/examples/coordinate_aspects.png",
```

Comment:
the legacy version of the documentation will stop being supported after a while and there is not much point in referring to it

You can view an analog in the production documentation or add an image to s3 bucket **nct-files/jpeg**